frontend/src/components/experimentRuns/ExperimentRunValidation.tsx

```diff
@@ -0,0 +1,231 @@
+							header={<Title level={5}>{`Lane ${lane.laneNumber}`}</Title>}
+							extra={getValidationStatusExtra(lane, isValidationInProgress)}
+						>
+							<LanePanel lane={lane} isValidationInProgress={isValidationInProgress} setAvailable={setAvailable} setPassed={setPassed} setFailed={setFailed}/>
+						</Collapse.Panel>
+					)
+				})
+			}</Collapse>
+		:
+			<Text italic style={{paddingLeft: '1em'}}>No datasets for this experiment are available for this experiment yet.</Text>
+	)
+}
+
+function FlexBar(props) {
+	// Displays children in a horizontal flexbox, maximizing the space between the children.
+	// Two children will appear at the left and right ends of the bar with whitespace in between.
+	return <div style={{ display: 'flex', justifyContent: 'space-between', padding: '1em' }}>{props.children}</div>
+}
+
+function getValidationStatusExtra(lane: LaneInfo, isValidationInProgress: boolean) {
+	switch (lane.validationStatus) {
+		case ValidationStatus.AVAILABLE: {
+			// return 
+			return (
+				<Space>
+					{isValidationInProgress ? <SyncOutlined spin/> : <QuestionCircleOutlined/>}
+					<Typography.Text strong>Needs validation</Typography.Text>
+				</Space>
+			)
+		}
+		case ValidationStatus.PASSED: {
+			return (
+				<Space>
+					{isValidationInProgress ? <SyncOutlined spin/> : <CheckOutlined style={{ color: 'green' }} />}
+					<Typography.Text strong>Passed</Typography.Text>
+				</Space>
+			)
+		}
+		case ValidationStatus.FAILED: {
+			return (
+				<Space>
+					{isValidationInProgress ? <SyncOutlined spin/> : <CloseOutlined style={{ color: 'red' }} />}
+					<Typography.Text strong>Failed</Typography.Text>
+				</Space>
+			)
+		}
+	}
+}
+
+interface LanePanelProps {
+	lane: LaneInfo
+	isValidationInProgress: boolean
+	setPassed: (lane: LaneInfo) => void
+	setFailed: (lane: LaneInfo) => void
+	setAvailable: (lane: LaneInfo) => void
+}
+
+function LanePanel({ lane, isValidationInProgress, setPassed, setFailed, setAvailable }: LanePanelProps) {
+
+	// Create a list of unique metrics url's from the lane's datasets. Normally all of the
+	// datasets should have the same url.
+	const urlSet: Set<string> = lane.datasets.reduce<Set<string>>((acc, dataset) => {
+		if (dataset.metricsURL) {
+			acc.add(dataset.metricsURL)
+		}
+		return acc
+	}, new Set<string>())
+
+	let title = 'Reads Per Sample'
+	if (lane.readsPerSample) {
+		title = `Reads Per Sample (${lane.readsPerSample.sampleReads.length})`
+	}
+
+	return (
+		<>
+			<FlexBar>
+				{urlSet.size > 0 && (
+					// Display the list of metrics url's associated with the lane's datasets.
+					<List>
+						{[...urlSet].map((url, index) => {
+							return (
+								<List.Item key={`URL-${index}`}>
+									<a href={url} rel="external noopener noreferrer" target="_blank">
+										View Metrics
+									</a>
+								</List.Item>
+							)
+						})}
+					</List>
+				)}
+
+				<Title level={5}>{title}</Title>
+				<Space>
+					<Button disabled={isValidationInProgress}
```

Comment:
I don't really like that the status can be flipped freely around. This makes it easy to set but it doesn't make the decision final ... We want to easily be able to see if a run was validated but this makes it possible to toggle it at a later time obscuring the original value at a given time to the user. Validation should not flip value every day. I would suggest 2 buttons (passed - failed) with modal confirmation. Reset would have to  be requested as a curation (or we could have the reset button be only visible to staff (us) for easy curation).